Week1/homework/app.js

```diff
@@ -1,11 +1,124 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
-  ];
+  const books = {
+    Peaks_and_Valleys_Making_Good_and_Bad_Times_Work_for_You: {
+      title: 'Peaks and Valleys: Making Good and Bad Times Work for You',
+      author: 'Spencer Johnson M.D',
+      language: 'English',
+      picture: 'Pictures/9200000035483858.jpg'
+    },
 
-  // Replace with your own code
-  console.log(bookTitles);
+    The_7_Habits_of_Highly_Effective_People: {
+      title: 'The 7 Habits of Highly Effective People: Powerful Lessons in Personal Change',
+      author: 'Stephen R. Covey',
+      language: 'English',
+      picture: 'Pictures/9200000041207941.jpg'
+    },
+
+    Who_Moved_My_Cheese: {
+      title: 'Who Moved My Cheese',
+      author: 'Spencer Johnson M.D',
+      language: 'English',
+      picture: 'Pictures/1001004000772743.jpg'
+    },
+
+    The_8th_Habit: {
+      title: 'The 8th Habit',
+      author: 'Stephen R. Covey',
+      language: 'English',
+      picture: 'Pictures/1001004002668308.jpg'
+    },
+
+    Everyday_Bias_identifying_and_Navigating_Unconscious_Judgments_in_Our_Daily_Lives: {
+      title: 'Everyday Bias identifying and Navigating Unconscious Judgments in Our Daily Lives',
+      author: 'Howard J. Ross',
+      language: 'English',
+      picture: 'Pictures/9200000026286353.jpg'
+    },
+
+    Thinking_Strategically: {
+      title: 'Thinking Strategically',
+      author: 'Harvard Business Review',
+      language: 'English',
+      picture: 'Pictures/9200000102594764.jpg'
+    },
+
+    Harvard_Business_Review_on_Winning_Negotiations: {
+      title: 'Harvard Business Review on Winning Negotiations',
+      author: 'Harvard Business Review',
+      language: 'English',
+      picture: 'Pictures/9200000002699149.jpg'
+    },
+
+    A_guide_to_the_Project_Management_Body_of_Knowledge: {
+      title: 'A guide to the Project Management Body of Knowledge',
+      author: 'Project Management Institute',
+      language: 'English',
+      picture: 'Pictures/9200000075545497.jpg'
+    },
+
+    Five_Minds_for_the_Future: {
+      title: 'Five Minds for the Future',
+      author: 'Howard Gardner',
+      language: 'English',
+      picture: 'Pictures/1001004006312039.jpg'
+    },
+
+    Blue_Ocean_Strategy: {
+      title: 'Blue Ocean Strategy',
+      author: 'W. Chan Kim',
+      language: 'English',
+      picture: 'Pictures/9200000025961603.jpg'
+    }
+  };
+
+  function convertBookToCard(newMainDev) {
+    const bookCollection = Object.values(books);
+    for (let i = 0; i < bookCollection.length; i++) {
+      const bookDiv = document.createElement('div');
+      // creates a main container for the image and the data
+
+      bookDiv.setAttribute('id', bookCollection[i].id);
```

Comment:
In general, you should not repeatedly use an expression like this: ` bookCollection[i]`. Instead, create a 'convenience' variable at the top of you loop body, for instance:

```js
const book = bookCollection[i];
```

and use that instead of repeating ` bookCollection[i]` in the rest of your code. It is less typing an reduces the chance of (typing) errors.